examples/example-ssl.rs

```diff
@@ -31,43 +29,46 @@ mod example {
             }
         };
 
-        // ~ OpenSSL offers a variety of complex configurations. Here is an example:
-        let mut builder = SslConnectorBuilder::new(SslMethod::tls()).unwrap();
-        {
-            builder.set_cipher_list("DEFAULT").unwrap();
-            builder.set_verify(SSL_VERIFY_PEER);
+        let rustls_config = {
+            let mut config = rustls::ClientConfig::new();
+            config
+                .root_store
+                .add_server_trust_anchors(&webpki_roots::TLS_SERVER_ROOTS);
             if let (Some(ccert), Some(ckey)) = (cfg.client_cert, cfg.client_key) {
                 info!("loading cert-file={}, key-file={}", ccert, ckey);
-
-                builder
-                    .set_certificate_file(ccert, X509_FILETYPE_PEM)
-                    .unwrap();
-                builder
-                    .set_private_key_file(ckey, X509_FILETYPE_PEM)
-                    .unwrap();
-                builder.check_private_key().unwrap();
+                let certs = {
+                    let file = fs::File::open(ccert).expect("cannot open private key file");
+                    let mut reader = BufReader::new(file);
```

Comment:
is there a special security centric reader or anything. Mostly some special way to mark that this memory contains privileged data or info? 
Might be overkill but I honestly don't know.

Reply:
Huh, never thought about it, do you have an example from another lang?